docs/introduction/abstract-filesystem.md

```diff
@@ -0,0 +1,32 @@
+# Abstract Filesystem
+One of the goals of the Shopsys Framework is to give you tools to make your e-commerce platform scalable. 
+One of the requirements for scalable application is separated file storage that is accessible from all application instances.
+We use abstract filesystem - [Flysystem](https://github.com/thephpleague/flysystem) via [OneUpFlysystemBundle](https://github.com/1up-lab/OneupFlysystemBundle) to fulfill this requirement. 
+
+## Flysystem
+[Flysystem](https://github.com/thephpleague/flysystem) allows you to easily swap out a local filesystem for a remote one like Redis, Amazon S3, Dropbox etc.
+
+### What is Flysystem used for
+In Shopsys Framework we currently use [Flysystem](https://github.com/thephpleague/flysystem) to store:
+- uploaded files and images
+- uploaded files and images via WYSIWYG
+- generated feeds
+- generated sitemaps
+
+### How to change storage adapter
+Flysystem supports a huge number of storage adapters. You can find [full list here](https://github.com/thephpleague/flysystem#community-integrations).
+You can change the setting of the adapter in `app/config/config.yml` file in `oneup_flysystem` section.
+There you find two adapters by default. 
+The first one `basic_filesystem` is the one that application uses for storing files. 
+In order to change the storage of your files update the `basic_filesystem` adapter appropriately.
+For more information how to set up adapter visit [OneUpFlysystem documentation](https://github.com/1up-lab/OneupFlysystemBundle/blob/master/Resources/doc/index.md#step3-configure-your-filesystems).
+
+### Create Nginx proxy to load files from different storage
+You have changed your storage and it works nicely. 
```

Comment:
Please change these two sentences to:
"If you change the file storage, you have to change also loading of these files to be accesible from the frontend of your application."